react-client/src/Components/DomInfoDialogBox/index.js

```diff
@@ -132,15 +134,16 @@ const DomInfoDialogBox = ({ id, idx, tag, classNames, parent, children, top, lef
           left: `${left}px`,
           border: `3px solid ${borderclr}`,
         }}
-      >
+      >        
         <button id="closedompeeker" className="close-btn-style" onClick={() => onClose(idx, id, uniqueID)}>
           x
         </button>
+       
         <div>
           <div className="dom-header">
             <span className="dom-header-tag">{tag}</span>
-            {id && <span className="dom-header-details">{id}</span>}
-            {domInfo.classNames.filter(obj => obj.name !== ".focused-dom").map((val) => (
+            {id && <span className="dom-header-details">{id}</span>}           
+            {domInfo.classNames.filter(obj => obj.name !== ".focused-dom"  && !obj.name.includes("custom-css")).map((val) => (              
```

Comment:
This is okay. But you can do the filtering inside domUtils instead. Have a look at this code snippet in line 78:

`const classNames = [...elTarget.classList].map((name) => `.${name}`);`